server/users/users.model.ts

```diff
@@ -32,4 +39,12 @@ export class UserModel implements DomainModel {
 		}
 	}
 
+	public getHashedIdentifier() {
+		return crypto.createHmac('sha512', this.salt!).update(this.id!.toString()).digest('hex');
```

Comment:
Yeah, it's deterministic
My line of thinking was: if we were to utilize UUIDs anywhere else, it would be extremely easy to generate unsubscribe links for other users maliciously.
This is why I wanted to combine it with some kind of salt, as a simple measure.
I think JWT-like approach, or something time limited (as you mentioned) is a great idea -- but is it overengineering for our current phase?
As a simple solution -- do you think we could simplify current setup `concat( hash(salt + user id), uuid)`
with something like `hash(salt + uuid)`?